scene/main/canvas_item.h

```diff
@@ -149,6 +149,10 @@ class CanvasItem : public Node {
 
 	void _notify_transform_deferred();
 
+#ifndef DISABLE_DEPRECATED
+	void _draw_circle_bind_compat_84472(const Point2 &p_pos, real_t p_radius, const Color &p_color);
```

Comment:
Move this next to the `_bind_compatibility_methods`, we normally group them

Reply:
done